{"frames":[{"func":"saveRDS(data_proc, \"input/data/data_proc.rds\")","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0},{"func":"gzfile(file, mode)","file":"","line_number":0,"end_line_number":0,"character_number":0,"end_character_number":0}],"message":"Error in gzfile(file, mode) : cannot open the connection\n"}